### PR TITLE
Create a simple CI + dependabot

### PR DESCRIPTION
**Pull request by modem7**
To make management easier, implement dependabot + create a basic docker CI to run against any PR's.

Also pin alpine version to allow for Dependabot to do version updates.

### CODE REVIEW

**Review by AndrewPaglusch — APPROVED** [review]
Looks great! Thank you for your contributions, @modem7. 

In the near future, I would like to push the built Docker image to the GitHub registry and update the README with instructions on how to use it. If that's something you would be open to contributing, it would be very much appreciated!

I'm going to hold off on merging this PR until one other maintainer reviews this in addition to my review.

**Comment by modem7** [comment]
> Looks great! Thank you for your contributions, @modem7. 
> 
> In the near future, I would like to push the built Docker image to the GitHub registry and update the README with instructions on how to use it. If that's something you would be open to contributing, it would be very much appreciated!
> 
> I'm going to hold off on merging this PR until one other maintainer reviews this in addition to my review.
> 
> 

I haven't yet looked at building to the github registry, but it shouldn't be too difficult all in all tbf! I'll see if I can get some time to look at it, but if anyone knows how to, then by all means!

**Review by mattburchett — APPROVED** [review]
Looks good to me!

**Pull request merged**
